scripts/docker/run-local.sh

```diff
@@ -6,4 +6,5 @@ else
     host_mountpoint=gd-fs
 fi
 
-docker run -i -t --privileged=true --rm -v "$host_mountpoint":/gd-fs gooddata-fs $1 $2 /gd-fs
+docker run -i -t --privileged=true --rm -v "$host_mountpoint":/gd-fs:shared gooddata-fs \
+    $(id -u) $(id -g) $1 $2 /gd-fs
```

Comment:
This will have problems on Mac, due to the intermediate VM layer. Allow setting the UID:GID manually or find a way how to determine automatically what user is running the VM.